biosimulations/apps/simulators/src/app/help/about/about.component.ts

```diff
@@ -9,7 +9,7 @@ export class AboutComponent implements OnInit {
   // TODO: get from app config
   apiUrl = 'https://api.biosimulations.org/'
```

Comment:
Here as well, I think it would make sense to portray each app as built and managed by the biosimulations team, with a common email/author

Reply:
We need to have separate team names and contact information. This is important for distinguishing these as separate resources. Its also important for enabling others to get involved in specific parts of the overall ecosystem without having to take on the whole thing. Going forward, the teams won't necessarily be identical. Arguably, there are already several more people tangentially involved with BioSimulators than BioSimulations -- James Feader, Ali Sinan Saglam, Chris Myers, Thomas Payne, Brian Drawert -- due to the need for help creating containers. We should do more to credit them on the website.

For example, many of the COMBINE projects (SBML, SED-ML, KISAO, SBO, COMBINE) are led by the same people. But because the teams don't completely overlap, its important to distinguish different teams.